src/ui/searchdialog.cc

```diff
@@ -211,21 +211,19 @@ qint64 SearchDialog::findPrevious(bool include_overlapping, bool interactive) {
   idx = lastIndexOf(_findBa, start_search_pos);
 
   if (idx >= 0) {
-    _hexEdit->setSelection(idx, _findBa.size(), true);
+    _hexEdit->setSelection(idx, _findBa.size(), /*set_visible=*/true);
     _lastFoundSize = _findBa.size();
-    emit enableFindPrevious(false);
-    //emit enableFindNext(true);
+    emit enableFindPrevious(true);
   } else {
     _lastFoundSize = 0;
     _hexEdit->setSelection(_hexEdit->dataModel()->binData().size() - 1, 0,
-                             false);
+                           /*set_visible=*/false);
```

Comment:
I added `/*set_visible=*/` here too. But it's on the next row. Maybe will be better add new variable?
```
qint64 size = _hexEdit->dataModel()->binData().size() - 1;
_hexEdit->setSelection(size, 0, /*set_visible=*/false);
```
I think it more readable code.

Reply:
Yup, seems like a good idea.
```C++
qint64 size = _hexEdit->dataModel()->binData().size() - 1;
```
Please use `auto` in such constructions, otherwise they are error-prone: in this case size_t (result of `.size()`) is silently converted to `qint64`. In current code it will be converted this way nonetheless (because that's the type of the first `setSelection()` argument), but if the argument type change someday (e.g. to size_t) then forced conversion through `qint64` may be problematic.

Reply:
Ok, I understand. I added it.